### ISSUE TITLE
Some stuff in the UWP version

### ISSUE THREAD

**Issue by Guerra24** [Drutol/MALClient#122]
New report will be added as a reply to this issue. Please do not close this issue.
UWP edition of #121.

Icons:
- 🐞 Bug - It doesn't do what it's supposed to do, required to fix (feature issues, crashes)
- 🐝 Bee - I want it to look better, not required to fix (ui, animation, etc)
- 🍄 Mushroom - Feature Request/Suggestion
- 💙 Blue Heart - I like it! General Feedback

Feel free to edit.

---
- 🐞 [Typo](https://puu.sh/vgp0A/86da3e55e7.png) in "sort values".
- 🐝 [Highlight](https://puu.sh/vgqeM/7026474e01.gif) is way too transparent. 
- 🍄 Move the [back arrow](https://puu.sh/vp2KH/4721c7d005.png) to the [window frame](https://puu.sh/vp2PF/7f9524a80b.png).
- 🍄 Images in articles should be cropped or scaled instead of stretched (it looks bad).
- 🍄 Bottom bar position; left is how looks now, right is how it should be. [Image](https://puu.sh/vp37L/f7711d3e4b.png)
- 🐝 The bottom bar changes color in the "Images" view.

**Comment by Drutol** [comment]
I guess I have to visit UWP now, so much to do so little time.

**Comment by Drutol** [comment]
I'll do this on Friday probably, where can I order 2nd pair of hands? ;d

**Comment by Drutol** [comment]
Update: well I've caught a cold and I was unable to do anything today :(

**Comment by Guerra24** [comment]
The Anime recommendations crash still happens, at least for me. [Vid](https://puu.sh/vv0eP/72557185eb.mp4)

**Comment by Guerra24** [comment]
- 🐞 Calendar highlight doesn't mach and the icon doesn't move with the background.  [Video](https://puu.sh/vvq1Q/61cc322b19.mp4)
- 🐞 Closing the settings doesn't restore to the previous selected item in the hamburger menu. [Video](https://puu.sh/vvqdC/a9f12f9e90.mp4)
- 🐞 Clicking an item multiple times will reload the view even if it is already loaded. [Video](https://puu.sh/vvqnO/94f9bedbbb.mp4)

**Comment by Drutol** [comment]
Does window size has anything to do with recommendations crashing?

**Comment by Guerra24** [comment]
No, maximized or any other size still crashes.

**Comment by Drutol** [comment]
And it changes to the different mode when it has more space before crash?

**Comment by Guerra24** [comment]
Yes, if maximized it shows the bottom bar.

**Comment by Drutol** [comment]
Hmm, is any of these devices yours?

![image](https://cloud.githubusercontent.com/assets/6102530/25418281/b88ebc80-2a4a-11e7-851d-e0aae7d31e7b.png)
![image](https://cloud.githubusercontent.com/assets/6102530/25418285/cc616dac-2a4a-11e7-95c0-2eb86cffea10.png)

**Comment by Guerra24** [comment]
No, I'm using a custom pc.

**Comment by Guerra24** [comment]
---
- 🐞 Pin news and articles crashes the app. [Video](https://puu.sh/vBwEm/d3f4c343fd.mp4)
- 🐞 Live tiles seem to not work (unconfirmed).

**Comment by Guerra24** [comment]
I would say that the UWP app is getting broken, since I have some free time I will start working on fixing this. I don't know much about UWP but have used C# in the past.
- 🐞 "Characters and staff" doesn't load images and crashes on click. [Video](https://puu.sh/vOdBQ/da25f69220.mp4)

**Comment by Drutol** [comment]
Yes, I've fixed this but I was waiting on feedback regarding freezes so I didn't push an update yet.

**Comment by Guerra24** [comment]
I mean, fixing all they issues listed here.

**Comment by Drutol** [comment]
Oh okay, good luck, getting some help would be huge booster ^^ I was on my phone so I couldn't edit it as I usually do.

**Comment by Guerra24** [comment]
The resizing crash... Visual Studio is unable to handle the exception, I'm unable to debug this crash. Looks that is something in the win32 side, way below in the system. SO, this bug is here from the beginning (most probably) or was introduced in an update.

**Comment by Guerra24** [comment]
UPDATE: The anime list system isn't the issue, it works correctly. There is something that gets triggered at a specific size. [Video1](), [Video2](). (Since my internet is a sh*t I can't upload the videos right now).

**Comment by Drutol** [comment]
Noooo please no... I've been fighting with this once for a long time... in the end it turned out that for some reason pivot with "Watching,Completed" etc was causing it when its index was set via binding. It was pure luck back then.

No wonder crash reporting isn't helpful...

**Comment by Guerra24** [comment]
And the "Anime recommendations" crash caused by an "Access Violation Exception" ![img](https://puu.sh/vQrrD/463ee320df.png)

**Comment by Drutol** [comment]
Pivot again...

**Comment by Drutol** [comment]
Try removing custom styles from these pivots?

**Comment by Drutol** [comment]
How is it going?

**Comment by Guerra24** [comment]
Now I hate visual studio, just saying xD. The VS project system is awful.

**Comment by Guerra24** [comment]
And yes, removing the custom styles fixes the crash.

**Comment by Drutol** [comment]
Well I don't see any issue with project structure in vs. For me the java one is much more messy. 

Vs is mediocre without ReSharper though.

I'll look into it today, thanks! ^^

**Comment by Drutol** [comment]
Hmm I was unable to check it out because "android open beta"... I will though as soon as I can

**Comment by Drutol** [comment]
It looks like MS changed pivot style completely with some update...

**Comment by Drutol** [comment]
I want to rewrite everything so badly now that I see some things that were here from day-one, god what monstrosity is this...

**Comment by Drutol** [comment]
Nothing has changed in HeaderItem which has these styles...

**Comment by Guerra24** [comment]
- 🐞 Filters doesn't work correctly with animes that doesn't have details loaded (season, start date, etc). [video](https://puu.sh/wkxFu/eae66ab089.mp4)

**Comment by Drutol** [comment]
They won't work because I don't have the data yet while arranging these items, once the data is here notice that they are sorted correctly.

**Comment by Guerra24** [comment]
mmm... if that's the case, it needs the "Load all details" button or triggering the "load details" when sorting.

**Comment by Drutol** [comment]
It would require too many requests, if user wants to have everything pulled he just needs to:
![image](https://user-images.githubusercontent.com/6102530/27169904-36728a86-51ac-11e7-9e78-00e71ed3cea4.png)

**Comment by Guerra24** [comment]
The "Load all details" is not available in search mode.
![img](https://puu.sh/wkzr8/ed202f4876.png)

**Comment by Drutol** [comment]
Hmm it was hidden as it was useless in certain contexts and I seem to have missed valid ones... Fixed

**Comment by Drutol** [comment]
I wonder whether MSFT fixed these issues with freezing somehow.

**Comment by Guerra24** [comment]
- 🐞 Missing delete button when adding new anime to list. [Video](https://puu.sh/woSsX/0c5a691387.mp4)

**Comment by Guerra24** [comment]
- 🐞 Season selector partially broken. [Video](https://puu.sh/wwhnd/a67494d3fb.mp4)

**Comment by Guerra24** [comment]
- 🐞 There is something broken causing a big memory leak. [Video](https://puu.sh/wAjJe/35608ece22.mp4)

**Comment by Drutol** [comment]
Wat. It always happens?

**Comment by Guerra24** [comment]
Yes, it's consistent. Go into plan to watch, then sort by season, scroll down and scroll up will trigger it.

**Comment by Drutol** [comment]
You just have broken computer... crashes on resize, memory leaks... it all works for me.

I don't know even where to start to fix it. Maybe you could ran memory profiling in visual studio. It has nice stats how many objects are allocated.

**Comment by Guerra24** [comment]
"It's all on my mind" xD. I'm going to do some testing and try on a friend's pc, just in case you're right.

**Comment by Guerra24** [comment]
- 🐞 Reviews text doesn't use the space correctly. [Video](https://puu.sh/wJceg/c104cfa64f.mp4)

**Comment by Guerra24** [comment]
@Drutol, What version of Visual Studio do you use?

**Comment by Drutol** [comment]
2017 Community (15.2)

**Comment by Guerra24** [comment]
- 🐞 Forum posts broke. ![img](https://puu.sh/xrZrr/d8612d0842.png)

**Comment by Drutol** [comment]
This is on new update?

**Comment by Guerra24** [comment]
Yes.

**Comment by Drutol** [comment]
Got it fixed.

**Comment by Guerra24** [comment]
- 🐝/🐞 Notify user in case of pending approval.

 ![gif](https://puu.sh/xRZ6A/483c98e9ae.gif)

**Comment by Drutol** [comment]
Hmm I'm not sure whether MAL returns generic error here or states the reason...

**Comment by Guerra24** [comment]
### Unexpected update, love it ❤️ 
![wtf](https://puu.sh/yfX1H/2c846c2ae5.png)

- 🐝 Why not integrate the top area (hamburger button, list name and anime name) into the titlebar?

**Comment by Drutol** [comment]
You're welcome :D

I really hate your brown accent colour ^^
Could be good idea, I'll try it out someday.

**Comment by Guerra24** [comment]
I've imported the project in VS but the shared project fails to compile, is there an extra step to make it compile?

![image](https://user-images.githubusercontent.com/9023392/32523170-cb6d35a0-c3df-11e7-9f6d-ea20e4292a21.png)

**Comment by Drutol** [comment]
#151 #165

**Comment by Guerra24** [comment]
Thanks! 👍

**Comment by Drutol** [comment]
Are you planning to blurify things?

**Comment by Guerra24** [comment]
Not really, do some minor maintenance. Fix a small typo that I can't unsee and check some stuff that doesn't work wright.

**Comment by Guerra24** [comment]
More problems... I'm unable to launch it from VS both in Debug and Release mode, but after deploying I can launch it manually. It gets stuck on splash forever, no error, nothing.

**Comment by Drutol** [comment]
Hmm what version of `Microsoft.NETCore.UniversalWindowsPlatform` nuget are you using?

**Comment by Guerra24** [comment]
![image](https://user-images.githubusercontent.com/9023392/32619724-2cd9a712-c541-11e7-9f2d-7a3a37901998.png)
😶

**Comment by Guerra24** [comment]
What do you think?

![image](https://user-images.githubusercontent.com/9023392/32644708-ff16c478-c5a8-11e7-92a0-df4c0dbc6788.png)
There is reveal in a lot of new places (I mean, A LOT).
![img](https://puu.sh/yiC7t/55cf85059e.png)
![image](https://user-images.githubusercontent.com/9023392/32644750-3381ae8a-c5a9-11e7-971d-442a78a74aac.png)
Switched to Acrylic blur for the header, increased image blur with a slightly darker color.
![image](https://user-images.githubusercontent.com/9023392/32644927-432ec4f2-c5aa-11e7-9e47-44abeb4c968d.png)

So.... 

![image](https://user-images.githubusercontent.com/9023392/32645084-073c902c-c5ab-11e7-8080-604e63068b4e.png) ![image](https://user-images.githubusercontent.com/9023392/32645054-e7ea1a3c-c5aa-11e7-8ed3-8271ee0c7944.png)

This change requires the UWP Shared code to use FCU SDK, I tried to use conditional Xaml but it requires CU SDK so...

Extra
---
Where do you set the title bar color?
Calendar is broke (this is since the latest store update).
Comp. view header text is also broke (this too).

**Comment by Drutol** [comment]
Wow, great job! 
Sorry that you had to go though this code ^^

I feel like the upper bar should be darker because now it's just "melting" with everything else.
https://github.com/Drutol/MALClient/blob/master/MALClient.Dekstop/App.xaml.cs#L190

I'll try to find some time today to fix airing stuff.

**Comment by Drutol** [comment]
Feel free to submit pull request :)

**Comment by Guerra24** [comment]
The entire app header (Title bar included)?

Okay 😃

**Comment by Drutol** [comment]
Probably? Depends on how it looks ^^

**Comment by Guerra24** [comment]
This is how it looks.

![image](https://user-images.githubusercontent.com/9023392/32647622-ca6a6c34-c5b7-11e7-8615-6290a562829e.png)

**Comment by Drutol** [comment]
Uppershadow looks nicer in my opinion.

Nice accent:
![image](https://user-images.githubusercontent.com/6102530/32650893-bad9f292-c5ff-11e7-9f7f-818320cb6534.png)

**Comment by Guerra24** [comment]
😂 I would prefer black but it can't be set so 🤷‍♂️ 

- I've

 tried to increase the Shared project min sdk to FCU but it complains about it no begin compatible with 10586, it needs that because I've used acrylic on the list elements, it works but I don't know if the store would accept that.

**Comment by Drutol** [comment]
I remember having some issues with that too. In the end I didn't increase it ^^
What matters most for the store is main project's configuration so it shouldn't complain.

**Comment by Guerra24** [comment]
Did you change something in the BGTasks? Now these projects refuse to compile 😐 

![img](https://puu.sh/yj6hP/02014c8742.png)

**Comment by Drutol** [comment]
![image](https://user-images.githubusercontent.com/6102530/32675305-c3044722-c656-11e7-91ac-303a163217a7.png)

**Comment by Guerra24** [comment]
Is there a way to style scrollbars? I've found a way to fix the ones that are behind acrylic by adding margin to top and bottom of the scrollbar.

**Comment by Drutol** [comment]
If I were you I wouldn't touch them. It's just pain. At least they were 1,5y ago.

**Comment by Guerra24** [comment]
It was NOT really hard, just added this:
![img](https://puu.sh/ykQ5U/078dd26e20.png)
[Here](https://github.com/Drutol/MALClient/blob/master/MALClient.Desktop/Pages/Main/AnimeListPage.xaml#L40) And...
![img](https://puu.sh/ykQ9g/9f166fd810.png)

**Comment by Guerra24** [comment]
Finally completed header integration into title bar. It frees up some space in the list and creates a 3-column-view on the entire window.
![img](https://puu.sh/ymmxT/e1f59ad5a4.png)

**Comment by Guerra24** [comment]
TL;DR Reworked again the UI.

![img]()
![img]()

**Comment by Guerra24** [comment]
Hey! I'm back after a month of inactivity. My PC's PSU broke and took a while to get a new one, I'm going to continue working on this on my free time but meanwhile I got a small maintenance PR.

**Comment by Drutol** [comment]
Hello! Thanks!

**Comment by Guerra24** [comment]
Got mobile fully working again, the PR is ready and both versions are OK to release (if nothing broke (I hope no)).

**Comment by Drutol** [comment]
Hehe, hmm... I haven't update mobile in ages... might be good idea ^^

**Comment by Guerra24** [comment]
It seems to have been fixed on its own... Feel free to merge.

**Comment by Guerra24** [comment]
I'm gonna change the target SDK to April Update (17134), are you OK with that?

**Comment by Drutol** [comment]
No clue, I didn't notice *There are new notifications* yet. (I'm on April's update)

Go ahead with new SDK, I'm fighting with MSFT about certification right now...

**Comment by Guerra24** [comment]
(Some) Promotional Videos broke, android and uwp affected.

Btw, because I'm gonna continue working here and to make this more easy for both, would you give me write access to the repo? So I don't have to make PRs for single commits or similar stuff.

**Comment by Drutol** [comment]
Well, I don't mind. The master will be kept protected though :)

**Comment by Guerra24** [comment]
Did some labels renames, examples: `Android` -> `Platform: Android`, `Info` -> `Type: Info`. And new lablels: `Platform: UWP`, `Status: Cannot Reproduce `, `Type: Bug`, `Type: Question`, etc.

Also write an [issue template](https://pastebin.com/GiLryisZ), what do you think?
